File: calculator.c
#include <stdio.h>

int main() {
    float a, b;
    char op;
    scanf("%f %c %f", &a, &op, &b);

    if(op == '+')
        printf("Result: %.2f\n", a + b);
    else if(op == '-')
        printf("Result: %.2f\n", a - b);
    else if(op == '*')
        printf("Result: %.2f\n", a * b);
    else if(op == '/') {
        if(b != 0)
            printf("Result: %.2f\n", a / b);
        else
            printf("Error: Division by zero\n");
    } else
        printf("Invalid operator\n");

    return 0;
}

File: calculator.java
import java.util.Scanner;

public class Calc {
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        double a = in.nextDouble();
        char op = in.next().charAt(0);
        double b = in.nextDouble();

        switch(op) {
            case '+': System.out.println("Result: " + (a + b)); break;
            case '-': System.out.println("Result: " + (a - b)); break;
            case '*': System.out.println("Result: " + (a * b)); break;
            case '/': 
                if(b != 0)
                    System.out.println("Result: " + (a / b));
                else
                    System.out.println("Error: Division by zero");
                break;
            default:
                System.out.println("Invalid operator");
        }
        in.close();
    }
}
